### PR TITLE
Fix too short dialog with the traverse info (closes #1441)

### LINKED ISSUE #1441: Traverse dialog has text that is out of bounds
## Bug description

image

Part of the text is out of bounds and there is no scroll to be able to see the text. Resizing the dialog does not help either. It seems to show up to me every time I click the T-in-circle icon in Git Machete.

For more non-trivial issues, we would also appreciate if you included the following details:
 - Git Machete plugin version: 3.3.0
 - IDE type: IntelliJ
 - IDE version: 2022.3.1 (Community Edition)
   ```
   Build #IC-223.8214.52, built on December 20, 2022
   Runtime version: 17.0.5+1-b653.23 amd64
   VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
   ```
 - Operating system: Ubuntu 22.04.1 LTS

### PR DESCRIPTION
**Pull request by mkondratek**

### CODE REVIEW

File: frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/traverse/BaseTraverseAction.java [hunk 1]
```diff
@@ -96,7 +96,7 @@ public void actionPerformed(AnActionEvent anActionEvent) {
             getString("action.GitMachete.BaseTraverseAction.dialog.traverse-approval.title"),
             getString("action.GitMachete.BaseTraverseAction.dialog.traverse-approval.text.HTML"),
             SHOW_TRAVERSE_INFO,
-            /* myHeight */ 100);
+            /* myHeight */ 120);
```
Comment (PawelLipski) [inline frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/traverse/BaseTraverseAction.java:99]: Huh just a 20% increase will be enough? 🤔

File: CHANGE-NOTES.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 ## v3.3.1
 - Fixed a race condition in the Slide-out action.
+- Fixed a too-short traverse info dialog.
```
Comment (PawelLipski) [inline CHANGE-NOTES.md:5]: Mini-nit
```suggestion
- Fixed a too short traverse info dialog.
```

File: frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/traverse/BaseTraverseAction.java (hunk 1 above)
Comment (mkondratek) [reply frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/traverse/BaseTraverseAction.java:99]: it's enough to me but I will perform beta testing 😆

**Review by PawelLipski — APPROVED** [review]

**Comment by mkondratek** [comment]
hm, we can do more and drop the infodialog class 😆

File: frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties [hunk 3]
```diff
@@ -113,7 +113,7 @@ action.GitMachete.BaseResetToRemoteAction.info-dialog.title=Git Machete Reset In
 action.GitMachete.BaseResetToRemoteAction.info-dialog.message.HTML=\
   <html>\
     Git Machete Reset will perform a reset of local branch <b>{0}</b> \
-    so that it will point to the same commit as its remote branch <b>{1}</b><nobr/>.<br><br> \
+    so that it will point to the same commit as its remote branch <b>{1}</b>.<br><br> \
```
Comment (mkondratek) [inline frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties:116]: without `<nobr>`:
image

with `<nobr>`:
image

File: frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties [hunk 4]
```diff
@@ -113,7 +113,7 @@ action.GitMachete.BaseResetToRemoteAction.info-dialog.title=Git Machete Reset In
 action.GitMachete.BaseResetToRemoteAction.info-dialog.message.HTML=\
   <html>\
     Git Machete Reset will perform a reset of local branch <b>{0}</b> \
-    so that it will point to the same commit as its remote branch <b>{1}</b><nobr/>.<br><br> \
```
Comment (PawelLipski) [inline frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties:116]: Huh maybe remove all my recently added `<nobr>` then while you're here? 😅 better to have some period or comma go to the next line, than have the line truncated like in your screen below 👇🏻

File: frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/traverse/BaseTraverseAction.java [hunk 5]
```diff
@@ -91,14 +93,13 @@ public void actionPerformed(AnActionEvent anActionEvent) {
     if (branchLayout != null && branchLayout.getRootEntries().nonEmpty() && gitRepository != null) {
       if (PropertiesComponent.getInstance(project).getBoolean(SHOW_TRAVERSE_INFO, /* defaultValue */ true)) {
 
-        val traverseInfoDialog = new InfoDialog(
```
Comment (PawelLipski) [inline frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/traverse/BaseTraverseAction.java:94]: :benissimo:

File: frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties (hunk 4 above)
Comment (mkondratek) [reply frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties:116]: _to ja cofam to co powiedziałem przed chwilą_

**Review by PawelLipski — APPROVED** [review]

**Pull request merged**